### PR TITLE
feat(BBD-658): change sayt products on highlight

### PR DESCRIPTION
**Pull request by effervescentia**

### CODE REVIEW

File: src/autocomplete/index.ts [hunk 1]
```diff
@@ -22,8 +32,8 @@ class Autocomplete {
     this.flux.on('sayt:select_active', this.selectActive);
   }
 
-  activationTargets() {
-    return this.root.querySelectorAll('.gb-autocomplete-target');
+  activationTargets(): NodeListOf<HTMLElement> {
+return <any>this.root.querySelectorAll('.gb-autocomplete-target');
```
Comment (Vic-Dev) [inline src/autocomplete/index.ts]: can this be one line?

**Review by Vic-Dev — CHANGES_REQUESTED** [review]
Looks good! one teeny tiny comment. also tests are failing?

File: src/autocomplete/index.ts (hunk 1 above)
Comment (effervescentia) [reply src/autocomplete/index.ts]: I've been trying to keep away from doing arrow functions in the components unless we need to in order to preserve `this` bindings

Comment (effervescentia) [reply src/autocomplete/index.ts]: tests are dependant on groupby/flux-capacitor#60

**Pull request merged**